src/UIComponents/ColumnInspector.jsx

```diff
@@ -77,9 +77,21 @@ class ColumnInspector extends Component {
           <form>
             <div>
               <label>
-                <div>
-                  {currentColumnData.id}: {currentColumnData.dataType}
-                </div>
+                <div>{currentColumnData.id}</div>
+                <select
+                  onChange={event =>
+                    this.handleChangeDataType(event, currentColumnData.id)
+                  }
+                  value={currentColumnData.dataType}
+                >
+                  {Object.values(ColumnTypes).map((option, index) => {
+                    return (
+                      <option key={index} value={option}>
+                        {option}
+                      </option>
+                    );
+                  })}
+                </select>
```

Comment:
If we have preloaded metadata, I think we should just show the data type here without allowing the user to change it.